modules/template/src/main/twirl/scaladex/view/topicsCloud.scala.html

```diff
@@ -0,0 +1,10 @@
+@import scaladex.core.model._
+
+@(
+  topics: Seq[TopicCount]
+)
```

Comment:
As for `ecosystemHighlights`, I would define this component in `frontpage.scala.html`.